tests/components/suez_water/test_sensor.py

```diff
@@ -0,0 +1,92 @@
+"""Test Suez_water sensor platform."""
+
+from datetime import timedelta
+from unittest.mock import MagicMock, patch
+
+from freezegun.api import FrozenDateTimeFactory
+from syrupy import SnapshotAssertion
+
+from homeassistant.components.suez_water.const import DATA_REFRESH_INTERVAL
+from homeassistant.components.suez_water.coordinator import PySuezError
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.const import STATE_UNAVAILABLE, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from . import setup_integration
+
+from tests.common import MockConfigEntry, async_fire_time_changed, snapshot_platform
+
+
+async def test_suez_sensors_valid_state(
+    hass: HomeAssistant,
+    snapshot: SnapshotAssertion,
+    suez_client: MagicMock,
+    mock_config_entry: MockConfigEntry,
+    entity_registry: er.EntityRegistry,
+) -> None:
+    """Test that suez_water sensor is loaded and in a valid state."""
+    with patch("homeassistant.components.suez_water.PLATFORMS", [Platform.SENSOR]):
+        await setup_integration(hass, mock_config_entry)
+
+    assert mock_config_entry.state is ConfigEntryState.LOADED
+    await snapshot_platform(hass, entity_registry, snapshot, mock_config_entry.entry_id)
+
+
+async def test_suez_sensors_failed_update(
+    hass: HomeAssistant,
+    suez_client,
+    mock_config_entry: MockConfigEntry,
+    freezer: FrozenDateTimeFactory,
+) -> None:
+    """Test that suez_water sensor reflect failure when api fails."""
+
+    with patch("homeassistant.components.suez_water.PLATFORMS", [Platform.SENSOR]):
```

Comment:
This one is not needed now

Reply:
You usually need it for the snapshot platform call, since that only snapshots one platform

Reply:
Ok I understant, thanks